lib/watch-files.js

```diff
@@ -1,17 +1,42 @@
 const chokidar = require('chokidar');
-const { isEmpty } = require('omnibelt');
+const { isEmpty, forEachSerialP } = require('omnibelt');
 const { logError, logResult, loadConfig, unlockConfig } = require('./utils');
 const paramsByCommand = require('./get-upload-params');
 const getUploader = require('./get-uploader');
 const path = require('path');
 
+const queueInfo = {
+  queue: [],
+  queueData: new Map(),
+  isRunning: false
+};
+
+setInterval(async () => {
```

Comment:
Keep the id returned in this setInterval and in the watcher error function I would [clear the interval](https://nodejs.org/api/timers.html#timers_clearinterval_timeout)

Reply:
And I would move this function into the returned function.